### ISSUE TITLE
PLUTON tokens not received in Ethereum Address/Wallet.

### ISSUE THREAD

**Issue by Nimeshbumb** [ethereum/mist#1203]
#### System information

```
Version: `0.8.3`
OS & Version: osx
Node type: `geth(default)`
```

If possible add the following to your report:
- Screenshots
  <img width="1028" alt="screen shot 2016-09-21 at 9 18 09 pm" src="https://cloud.githubusercontent.com/assets/9002185/18811649/62fba1f6-826c-11e6-978a-d39536d320e3.png">
- Check the console, of Mist (`CTRL/CMD + ALT + i`) and take a screenshot

<img width="555" alt="screen shot 2016-09-24 at 3 36 23 pm" src="https://cloud.githubusercontent.com/assets/9002185/18811663/b3c48ba2-826c-11e6-87a3-513c4e9681ac.png">
- Log files (Go to `menu -> accounts -> backup -> application data` and upload as zip-archive):

[Ethereum Wallet.zip](https://github.com/ethereum/mist/files/491384/Ethereum.Wallet.zip)

**Comment by Nimeshbumb** [comment]
etherscan.io shows that the Plutons have been sent to my ethereum address but I can't see those in my ethereum wallet since the Custom Token isn't accepting the Pluton Address! Can anyone look into this? Thanks!

**Comment by luclu** [comment]
@Nimeshbumb judging from the greyed out Token in your screenshot, it seems like your node is not fully synced with the network or you are on a different network. 
What network and block are you currently on?
Also feel free to check the your token balance via https://etherscan.io/token/Pluton.

**Comment by Nimeshbumb** [comment]
It shows sync with latest block and what do you mean by completely
different network.? Different ETH Address you mean?

**Comment by luclu** [comment]
What your current blocknumber?

**Comment by Nimeshbumb** [comment]
Also, it is stuck now at this screen. 



I have to forcefully say launch application every time.

**Comment by Nimeshbumb** [comment]
New Blocknumber..

**Comment by luclu** [comment]
As there are several issues with geth `<=1.4.12` please try to update geth to version `1.4.13`.
https://github.com/ethereum/mist/issues/1195#issuecomment-249547854:
> There's a new attack ongoing. It has been averted in Geth 1.4.13. If you're running Linux or OSX, you can replace your bundled geth from the wallet with one from https://github.com/ethereum/go-ethereum/releases/tag/v1.4.13 . Windows builds are coming soon too.

You can download and replace the binary of geth within Ethereum Wallet:
```
$ cd ~/Downloads
$ wget https://github.com/ethereum/go-ethereum/releases/download/v1.4.13/geth-OSX-20160926130115-1.4.13-8f0db697.zip
$ unzip geth-OSX-20160926130115-1.4.13-8f0db697.zip
$ mv geth-OSX-20160926130115-1.4.13-8f0db697 /Applications/Ethereum\ Wallet.app/Contents/nodes/geth/mac-x64/
```
Or wait for the upcoming fix-release `0.8.4`.

**Comment by evertonfraga** [comment]
Please refer to [this issue](https://github.com/ethereum/mist/issues/3097) for more information and our [Wiki](https://github.com/ethereum/mist/wiki) for syncing troubleshooting steps.
